ScrutinyPlot/mkscrutinydatafiles.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+datadir=/afs/cern.ch/work/c/cvuosalo/popularity/
```

Comment:
Script should accept input datadir rather than rely on hardcoded value. Please change.